src/PlottingInterface_TEST.cc

```diff
@@ -126,6 +190,10 @@ TEST(PlottingInterfaceTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(Transport))
 
   node.Subscribe("collision_topic", cb);
 
+  auto topics = transport.Topics();
+  double timeRef = 10;
+  topics["/collision_topic"]->SetPlottingTimeRef(&timeRef);
```

Comment:
Here you're passing a reference to an object that will be destroyed soon. This is one of the reasons to use a smart pointer instead.